### PR TITLE
Add examples of authenticator types to Authenticator definition

### PR DESCRIPTION
**Pull request by emlun**
This would merge into #1130. Fixes the rest of #1128.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/pull/1131.html" title="Last updated on Feb 27, 2019, 6:12 PM UTC (a68031a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1131/ca62975...a68031a.html" title="Last updated on Feb 27, 2019, 6:12 PM UTC (a68031a)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -553,6 +553,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     cryptographically signing and returning, in the form of an [=Authentication Assertion=], 
     a challenge and other data presented by a [=[WRP]=] (in concert with the [=[WAC]=]).
 
+    A [=[WAA]=] could be, for example, a dedicated USB or wireless device separate from the [=client device=],
+    a dedicated hardware subsystem integrated into the [=client device=],
+    or a software component of the [=client=].
+    [=Authenticators=] can report their type and security characteristics via [=attestation=].
```
Comment (selfissued) [inline index.bs]: I believe that, grammatically, "attestation" should be "attestations".

Comment (emlun) [reply index.bs]: I think this sentence can be argued to be using "attestation" as an uncountable noun, as is also done in the [Attestation](https://w3c.github.io/webauthn/#attestation) definition:

>[...] In the WebAuthn context, attestation is employed to attest to the provenance of an authenticator and the data it emits; [...]

Do you agree?

**Comment by nadalin** [comment]
@equalsJeffH Please review

**Comment by emlun** [comment]
@equalsJeffH Good point. Maybe just a "see [Authenticator Taxonomy] for details"?

File: index.bs (hunk 1 above)
Comment (emlun) [reply index.bs]: Moving ahead with merge per @nadalin's request

**Pull request merged**